pkg/app/create.go

```diff
@@ -68,6 +68,11 @@ var CommandCreate = &cli.Command{
 			Value:   sshconfig.GetPrivateKeyOrPanic(),
 			Hidden:  true,
 		},
+		&cli.StringFlag{
+			Name:  "host",
```

Comment:
I am wondering if we should make it as a CLI argument, or keep it in build.envd.

WDYT

Reply:
I took a look at the section of `lang.configs`. I think ssh port listening is like a behaviour of enviroment not applications in environment. Secondly, If users want to use this function, mostly because they want to share this environment or use it in remote server. Both seems to be a operation maintain purpose not application modified requirement.

Reply:
That is why I decide to move this function into cli.

Reply:
SGTM.

/cc @kemingy @VoVAllen